plugins/outputs/bigquery/sample.conf

```diff
@@ -4,7 +4,7 @@
   credentials_file = "/path/to/service/account/key.json"
 
   ## Google Cloud Platform Project
-  project = "my-gcp-project"
+  # project = "*detect-project-id*"
```

Comment:
Please add a comment to the project parameters that describes what the special use of `*detect-project-id*` does (i.e. that this is a special string recognized by the library to auto-detect the library) and any other string is used as the project name.

Reply:
Should I maybe revert that and indicate that the empty string can be used to auto-detect the project?

Reply:
@srebhan had a similar remark? https://github.com/influxdata/telegraf/pull/14086#discussion_r1352801763

Reply:
I like the idea of an empty string defaults to the auto-detect! I still think a comment detailing that would be good as well.

Reply:
Sure, was going to add that anyway. But it might make more sense for telegraf users to just leave the `project` empty..